### PR TITLE
[DOCS] Updating contributing section and fixing capitalisation

### PR DESCRIPTION
**Pull request by franzihei**

### CODE REVIEW

File: docs/contributing.rst [hunk 1]
```diff
@@ -9,14 +9,14 @@ In particular, we appreciate support in the following areas:
...
-  `good first issue <https://github.com/ethereum/solidity/labels/good%20first%20issue>`_ which are
+  `"good first issue" <https://github.com/ethereum/solidity/labels/good%20first%20issue>`_ which are
   meant as introductory issues for external contributors.
 * Improving the documentation.
 * Translating the documentation into more languages.
 * Responding to questions from other users on `StackExchange
   <https://ethereum.stackexchange.com>`_ and the `Solidity Gitter Chat
   <https://gitter.im/ethereum/solidity>`_.
-* Getting involved in the language design process by joining language design calls, proposing language changes or new features and providing feedback.
+* Getting involved in the language design process by proposing language changes or new features in the forum and providing feedback.
```
Comment (hrkrshnn) [inline docs/contributing.rst]: Maybe we could link the forum here as well?

File: docs/contributing.rst [hunk 2]
```diff
@@ -460,17 +459,26 @@ needed for documentation and checks for any problems such as broken links or syn
...
-where existing properties of the language and proposals for new features can be discussed.
+To actively get involved in the language design process and share your ideas concerning the future of Solidity,
+please join the `Solidity forum <https://forum.soliditylang.org/>`_, where existing properties of the language
+and proposals for new features can be discussed.
 
-We regularly host language design discussion calls, in which selected topics, issues or feature implementations are debated in detail. The invitation
-to those calls is shared via the aforementioned forum. We are also sharing feedback surveys and other language design relevant content in this forum.
+In the ``language design`` category, you can propose and discuss new language features and their implementation in
+the early stages of ideation or modifications of existing features. The forum serves as the place to discuss the
+initial soundness of the ideas/proposals and to define them further. As soon as proposals get more tangible their
```
Comment (hrkrshnn) [inline docs/contributing.rst]:
```suggestion
initial soundness of the ideas/proposals and to define them further. As soon as proposals get more tangible, their
```

Comment (hrkrshnn) [reply docs/contributing.rst]: This paragraph reads slightly complicated. Maybe the first line can be just:


> In the ``language design`` category, you can propose and discuss new language features or modifications of existing features.

followed by

> The forum serves as a place to discuss proposals and refine them.

Yeah, but this is just subjective, anyways.

**Comment by franzihei** [comment]
Thanks @hrkrshnn :) Implemented your feedback and tried to make the lang design wording easier.

**Comment by chriseth** [comment]
Looks good, but please squash before merge.

File: docs/contributing.rst [hunk 3]
```diff
@@ -460,17 +459,26 @@ needed for documentation and checks for any problems such as broken links or syn
...
-For ad-hoc cases and questions you can reach out to us via the `Solidity-dev Gitter channel <https://gitter.im/ethereum/solidity-dev>`_, a
-dedicated chatroom for conversations around the Solidity compiler and language development.
+As soon as proposals get more tangible, their
+implementation will also be discussed in the `Solidity GitHub repository <https://github.com/ethereum/solidity>`_
+in the form of issues.
+
+In addition to the forum and issue discussions, we regularly host language design discussion calls, in which selected
+topics, issues or feature implementations are debated in detail. The invitation to those calls is shared via the forum.
+
+We are also sharing feedback surveys and other language design relevant content in the forum.
```
Comment (hrkrshnn) [inline docs/contributing.rst]:
```suggestion
We are also sharing feedback surveys and other content, relevant to language design in the forum.
```

or maybe?

```suggestion
We are also sharing feedback surveys and other language design contents in the forum.
```

**Review by hrkrshnn — DISMISSED** [review]

**Review by hrkrshnn — APPROVED** [review]

**Pull request merged**